lib/tapioca/helpers/rbi_files_helper.rb

```diff
@@ -250,5 +248,14 @@ def update_gem_rbis_strictnesses(errors, gem_dir)
     def gem_name_from_rbi_path(path)
       T.must(File.basename(path, ".rbi").split("@").first)
     end
+
+    sig { params(block: T.proc.void).returns(Numeric) }
+    def with_timer(&block)
```

Comment:
Could we use `Benchmark` instead? I think it does exactly the same. Also, do we actually need to measure this?

Reply:
> Could we use Benchmark instead? 

Good idea!

> Also, do we actually need to measure this?

I was actually thinking to extend the concept to other commands so we can easily see where Tapioca is slow or slower.